core/src/main/resources/error/error-classes.json

```
@@ -1816,6 +1843,11 @@
     ],
     "sqlState" : "42883"
   },
+  "SCALAR_SUBQUERY_IS_IN_GROUP_BY_OR_AGGREGATE_FUNCTION" : {
+    "message" : [
+      "The correlated scalar subquery '<sqlExpr>' is neither present in GROUP BY, nor in an aggregate function. Add it to GROUP BY using ordinal position or wrap it in `first()` (or `first_value`) if you don't care which value you get."
```

Please, remove quotes around <sqlExpr>:
```suggestion
      "The correlated scalar subquery <sqlExpr> is neither present in GROUP BY, nor in an aggregate function. Add it to GROUP BY using ordinal position or wrap it in `first()` (or `first_value`) if you don't care which value you get."
```